core/gui/src/app/workspace/service/workflow-graph/model/joint-graph-wrapper.ts

```diff
@@ -787,48 +700,6 @@ export class JointGraphWrapper {
     this.jointLinkUnhighlightStream.next([linkID]);
   }
 
-  /**
-   * Show the breakpoint button of a given link
-   * emits an event to the link breakpoint show stream.
-   * @param linkID
-   */
-  public showLinkBreakpoint(linkID: string): void {
-    if (!this.linksWithBreakpoints.includes(linkID)) {
-      this.linksWithBreakpoints.push(linkID);
-    }
-    this.jointLinkBreakpointShowStream.next({ linkID });
-  }
-
-  /**
-   * Hide the breakpoint button of a given link
-   * emits an event to the link breakpoint hide stream.
-   * @param linkID
-   */
-  public hideLinkBreakpoint(linkID: string): void {
-    if (!this.linksWithBreakpoints.includes(linkID)) {
-      return;
-    }
-    const LinkIndex = this.linksWithBreakpoints.indexOf(linkID);
-    this.linksWithBreakpoints.splice(LinkIndex, 1);
-    this.jointLinkBreakpointHideStream.next({ linkID });
-  }
-
-  /**
-   * This method resizes the element according to given width and height.
-   * An element can be an operator or a group.
-   */
-  public setElementSize(elementID: string, width: number, height: number): void {
-    const cell: joint.dia.Cell | undefined = this.jointGraph.getCell(elementID);
-    if (!cell) {
-      throw new Error(`element with ID ${elementID} doesn't exist`);
-    }
-    if (!cell.isElement()) {
-      throw new Error(`${elementID} is not an element`);
-    }
-    const element = <joint.dia.Element>cell;
-    element.resize(width, height);
-  }
```

Comment:
those are not related to group?

Reply:
`showLinkBreakpoint` and `hideLinkBreakpoint` are unused functions, and `setElementSize` became unused after the `repositionGroup` function was removed.